openapi/docusign.admin/openapi.yaml

```diff
@@ -0,0 +1,5083 @@
+openapi: 3.0.1
+info:
+  title: DocuSign Admin API
+  description: An API for an organization administrator to manage organizations, accounts and users which enables you to automate user management with your existing systems while ensuring governance and compliance.
```

Comment:
Shall we follow the standard used in other connectors in the description as follows
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/ably/openapi.yaml#L14
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/spotify/openapi.yaml#L7